server/server.go

```diff
@@ -49,6 +49,7 @@ type config struct {
 	EnableDebugger   bool   `envconfig:"ENABLE_DEBUG" default:"false"`
 	EnableTestExport bool   `envconfig:"ENABLE_TEST_EXPORT" default:"true"`
 	KeployApp        string `envconfig:"APP_NAME" default:"Keploy-Test-App"`
+	PORT             string `envconfig:"PORT" default:"6789"`
```

Comment:
@Pradhyuman-sharma please follow the naming convention of the other variables. The go variable should be `Port` and not `PORT`